src/content/getting-started/cloud-provider-accounts/cluster-api/azure/index.md

```diff
@@ -0,0 +1,136 @@
+---
+linkTitle: Azure
+title: Prepare an Azure subscriptions to run Cluster API Giant Swarm cluster
+description: This guide will walk you through all necessary steps to set up an Azure subscriptions with appropriate IAM roles for operating Cluster API Giant Swarm clusters.
+menu:
+  main:
+    identifier: gettingstarted-cloudprovider-clusterapi-azure
+    parent: gettingstarted-cloudprovider-clusterapi
+weight: 20
+user_questions:
+  - How do I prepare my Azure subscription for Cluster API?
+  - What do I need to configure in Azure in order to run Cluster API Giant Swarm clusters?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-10-16
+---
+
+Currently the Giant Swarm Cluster API for Azure implementation supports only running the Management and Workload Clusters in a single subscription.
+
+## Overview
+
+This document consists of the instructions for setting up Azure subscriptions that are needed to run Cluster API for Azure with Giant Swarm.
+
+## Procedure for Azure subscription configuration
+
+Currently implemented Management Clusters bootstrap process [`mc-bootstrap`](https://github.com/giantswarm/mc-bootstrap)  with Cluster API for Azure (CAPZ) requires a `bootstrap Service Principal` to authenticate the `local capz` running in `kind` onto Azure. This is required to create the final result, that is the Management Cluster in the cloud provider and delegate the `capz controller responsabilities` to it.
+
+After the initial bootstrap is completed the `capz-controller` will use a `User Assigned managed identity` , created during the `mc-bootstrap` process for all further cloud interaction.
+
+This identity can:
+
+* Create Workload Clusters , and all the related resources, in the `same subscription` where the MC lives
+* Be `authorized by the Customer` to create Workload Clusters, and all the related resources, in _other subscriptions_ belonging to the same tenant - **NOTE** _this is not yet implemented_
+* Be `authorized by the Customer` to create Workload Clusters, and all the related resources, in _subscriptions that exist in other tenants_ - **NOTE** _this is not yet implemented_
+
+### 1. Prerequisites
+
+To create and assign the role to Giant Swarm's Service Principal you need:
+
+* An account with [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [User Access Administrator](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#user-access-administrator) role.
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) installed.
+
+### 2. Create management cluster service principal for bootstrapping
+
+First step is to create a `subscription` to host the `MC Cluster` and a `bootstrap SP` that will be used for the initial creation of the Cluster.
+
+#### Step 1a - Create the `bootstrap sp` using AZ CLI
```

Comment:
Above these choices a vs. b, we should state that either CLI or portal can be used.